### ISSUE TITLE
Can I ask the problem about the custom text data? Do the train, test, val data use the same csv file?

### ISSUE THREAD

**Issue by wusuhuang** [asigalov61/Torah-seq2seq-PyTorch#1]
![image](https://user-images.githubusercontent.com/57821107/161358302-ef416ec5-74ec-477e-9688-0706c4034111.png)

**Comment by asigalov61** [comment]
@wusuhuang Yes. In the case of Torah, there is nothing else that you can validate upon (unfortunately). However, for other datasets, you can change it to whatever you like.

Here is the original code/notebook that can give you all possible information about it, including the explanations:
https://github.com/bentrevett/pytorch-seq2seq/blob/master/6%20-%20Attention%20is%20All%20You%20Need.ipynb

Hope this helps :)
